ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host2/daemon-terminattr.yml

```
@@ -1,5 +1,10 @@
-### Daemon TerminAttr
+---
 daemon_terminattr:
+  cvaddrs:
+    - apiserver.arista.io:443
+  cvauth:
+    method: "key"
```

you can remove all the quotes around the strings
```suggestion
    method: key
```

(and below)

---

Removed for all.